tutorials/ab3AddRemark.md

```diff
@@ -141,6 +165,47 @@ public class RemarkCommand extends Command {
 }
 ```
 
+<panel header="Update construction of `RemarkCommand` in `RemarkCommandTest`">
+
+- Since `RemarkCommand`’s constructor now accepts arguments, the tests will reflect that.
```

Comment:
Since both the constructor and execute method of `RemarkCommand` was updated, how about phrasing this like this instead?

Since `RemarkCommand` now requires both an index and a remark for instantiation, the tests should be updated accordingly. Additionally, the execution of the command has been updated, so the tests should be adjusted to reflect these changes.

```java
@Test
public void execute() {
   final String remark = "Some remark";

   assertCommandFailure(new RemarkCommand(INDEX_FIRST_PERSON, remark), model,
           String.format(MESSAGE_ARGUMENTS, INDEX_FIRST_PERSON.getOneBased(), remark));
}
```